backend/src/main/java/com/bakdata/conquery/io/result/ResultRender/ResultRenderProvider.java

```diff
@@ -0,0 +1,26 @@
+package com.bakdata.conquery.io.result.ResultRender;
+
+import com.bakdata.conquery.commands.ManagerNode;
+import com.bakdata.conquery.io.cps.CPSBase;
+import com.bakdata.conquery.models.execution.ManagedExecution;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import io.dropwizard.jersey.setup.JerseyEnvironment;
+
+import javax.ws.rs.core.UriBuilder;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.Optional;
+
+@CPSBase
+@JsonTypeInfo(use = JsonTypeInfo.Id.CUSTOM, property = "type")
+public interface ResultRenderProvider {
+
+	/**
+	 * Determines if by default the result url for this provider should be hidden or not
+	 */
+	boolean isHidden();
+
+	Optional<URL> generateResultURL(ManagedExecution<?> exec, UriBuilder uriBuilder);
```

Comment:
eigentlich kannst du dir die logik die du oben machst sparen wenn du hier allRenderers reingibst? Dann muss niemand außer der Klasse was von isHidden wissen.

Reply:
ich habe den test reingezogen in die provider